### DISCUSSION TITLE
Does the new figma connect plugin support self deployed storybook?

### DISCUSSION THREAD

**Discussion by DamianPereira** [storybookjs/storybook#18282] [Help]
I'm not sure from reading the documentation for https://www.figma.com/community/plugin/1056265616080331589/Storybook-Connect or the blog posts if my use case would be possible. If I have storybook deployed to a vps of my own, and available to the world wide web, can I then link figma to that storybook urls? Or it needs to be inside chromatic? 

The repositories I'm trying to use are behind a vpn, so it is not possible to use a free chromatic account for this, must I use an enterprise account for this usecase?

**Comment by shawn-wm** [comment]
We're also looking for an answer to @DamianPereira's question. If it is not natively supported via the Storybook Connect plugin for Figma (which appears to *only* leverage Chromatic), perhaps there is an adequate workaround to link to a self-hosted Storybook?

**Comment by domyen** [comment]
Hey there, maintainer here. It needs to be inside Chromatic (this enables the Storybook urls to sync with a db and get read by different instances of the plugin).
